core/chains/evm/config/toml/defaults/Hedera_Mainnet.toml

```diff
@@ -0,0 +1,33 @@
+ChainID = '295'
+ChainType = 'hedera'
+#considering the 3-5 (6 including a buffer) seconds of finality and 2 seconds block production
+#we set the depth to 6/2 = 3 blocks, setting to 10 for safety
+FinalityDepth = 10
+#Hedera has high TPS, so polling less often 
+LogPollInterval = '10s'
+MinIncomingConfirmations = 1
+#turning off the liveness detection, since the Websocket connection is not reliable, can be turned on if needed
+NoNewHeadsThreshold = '0'
+
+[BalanceMonitor]
+Enabled = true
+
+[GasEstimator]
+Mode = 'SuggestedPrice'
+# since Hedera dont have mempool and there's no way for a node to front run or a user to bribe a node to submit the transaction earlier than it's consensus timestamp
+# but they have automated congesting pricing throttling which would mean at high sustained level the gasPrice itself could be increased to prevent malicious behaviour.
+# # disabling the Bumpthreshold as TXM now implicity handles the bumping after checking on-chain nonce & re-broadcast for Hedera chain type
+BumpThreshold = 0
+BumpMin = '10 gwei'
+BumpPercent = 20
+
+[Transactions]
+# to hit throttling you'd need to maintain 15 m gas /sec over a prolonged period of time.
+# Because Hedera's block times are every 2 secs it's less less likely to happen as compared to other chains
+# Setting this to little higher even though Hedera has High TPS, We have seen 10-12s to get the trasaction mined & 20-25s incase of failures
+ResendAfterThreshold = '1m'
+
+
+[NodePool]
+#Disabling sync threshold check for testing
```

Comment:
Seems like this was disabled for testing. Should be set it to something meaningful now?

@dhaidashenko ?

Reply:
Yes, it's better to set it to something meaningful now. If we are running on single RPC it won't affect it's health.

Reply:
again with single RPC & unreliable sync times, setting this throws a lot of error & warning & marks it out-of-sync.
Rather than setting this to a really high number, we disabled it until we get more stable RPCs to work with